test/pxenforcer.test.js

```diff
@@ -100,7 +100,8 @@ describe('PX Enforcer - pxenforcer.js', () => {
         const reqStub = sinon.stub(request, 'get').callsFake((data, config, callback) => {
             callback(null, { headers: { 'x-px-johnny': '1' }, body: 'hello buddy', proxy: '' });
         });
-        req.originalUrl = '/_APP_ID/init.js';
+        req.path = '/_APP_ID/init.js';
+        req.originalUrl = `${req.protocol}://${req.hostname}${req.path}`;
```

Comment:
The property `req.originalUrl` used to be just the path, but now it's the entire URL. Is this a breaking change?

Reply:
I think this is a problem because it comes from Express / Node's built-in HTTP module: https://expressjs.com/en/api.html#req.originalUrl

Reply:
Not sure I understand.
Let's zoom about it I think it's important